plugins/source/aws/docs/tables/aws_docdb_cluster_snapshot_attributes.md

```diff
@@ -0,0 +1,17 @@
+# Table: aws_docdb_cluster_snapshot_attributes
+
+https://docs.aws.amazon.com/documentdb/latest/developerguide/API_DBClusterSnapshotAttributesResult.html
+
+The primary key for this table is **_cq_id**.
+
+
+## Columns
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id (PK)|UUID|
+|_cq_parent_id|UUID|
+|db_cluster_snapshot_arn|String|
+|db_cluster_snapshot_attributes|JSON|
```

Comment:
Sorry, looking at this again... I think if there's only this one column in this table, and this table maps to cluster snapshots one:one, then we should rather add `attributes` as a JSON column to `aws_docdb_cluster_snapshots`, similar to how we would add tags as a JSON column despite needing another SDK call.